lamindb_setup/dev/_hub_core.py

```diff
@@ -208,7 +208,7 @@ def _load_instance(
     return instance, storage
 
 
-def access_aws(access_token: Optional[str] = None) -> Dict[str, str]:
+def access_aws(storage_root: str, access_token: Optional[str] = None) -> Dict[str, str]:
     from .._settings import settings
 
     if settings.user.handle != "anonymous" or access_token is not None:
```

Comment:
`storage_root` is not passed below, but actually should, i think.

Reply:
`credentials = call_with_fallback_auth(_access_aws, storage_root=storge_root, access_token=access_token)`

Reply:
Right!

Reply:
It got removed resolving conflicts with main.